.github/workflows/testing.yml

```
@@ -14,13 +14,14 @@ jobs:
       fail-fast: false
       matrix:
         os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
-        python-version: ['3.6', '3.7', '3.8', '3.9']
+        python-version: ['3.7', '3.8', '3.9', '3.8', '3.9', '3.10', '3.11']
```

```suggestion
        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```
probably what you meant :)

---

Oops, nice catch, thanks.